src/Symfony/Component/Validator/README.md

```diff
@@ -7,7 +7,7 @@ The Validator component provides tools to validate values following the
 Resources
 ---------
 
-  * [Documentation](https://symfony.com/doc/current/book/validation.html)
+  * [Documentation](https://symfony.com/doc/current/components/validator.html)
```

Comment:
I would have used http://symfony.com/doc/current/validation.html instead which shows the integration into the full-stack framework (like the old document did).